internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,62 @@
 package daysteps
 
 import (
-	...
+	"Final-task-5/internal/personaldata"
+	"Final-task-5/internal/spentenergy"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
 )
 
 const (
 	StepLength = 0.65
 )
 
 // создайте структуру DaySteps
-...
-
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
 
 // создайте метод Parse()
-...
+func (ds *DaySteps) Parse(datastring string) (err error) {
+	slice := strings.Split(datastring, ",")
+
+	if len(slice) != 2 {
+		return errors.New("Длина слайса != 2")
+	}
+	steps, err := strconv.Atoi(slice[0])
+	if err != nil {
+		return errors.New("Ошибка преобразования string в int")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return fmt.Errorf("conversion error: %w", err)
```